package.json

```diff
@@ -29,6 +27,8 @@
     "postcss": "^8.2.15",
     "postcss-cssnext": "^3.1.0",
     "tailwindcss": "^2.1.2",
+    "webpack": "^4.46.0",
```

Comment:
Could we also remove these changes as well from the PR? I'am not sure why these needed to be changed?